### PR TITLE
Added new caption for jared-sanders slide

### PR DESCRIPTION
**Pull request by jared-sanders**

### CODE REVIEW

File: _slides/09-jared-sanders.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This is a caption!
```
Comment (beardofedu) [inline _slides/09-jared-sanders.md:6]: Hey , this is great!

File: _slides/09-jared-sanders.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
 ![codercat](https://octodex.github.com/images/codercat.jpg)
```
Comment (beardofedu) [inline _slides/09-jared-sanders.md:8]: If you want to change your image, you can grab a new one from here: https://octodex.github.com

File: _slides/09-jared-sanders.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _slides/09-jared-sanders.md]:
```suggestion
title: "Welcome to jared-sanders' slide!"
```

**Review by beardofedu — APPROVED** [review]
Summary comment of my great review

**Review by beardofedu — CHANGES_REQUESTED** [review]
im requesting changes

File: _slides/09-jared-sanders.md (hunk 2 above)
Comment (jared-sanders) [reply _slides/09-jared-sanders.md:8]: No thanks 👍

**Review by beardofedu — APPROVED** [review]
👍

**Pull request merged**